subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/maven/MavenLocalRepoResolveIntegrationTest.groovy

```diff
@@ -64,7 +64,7 @@ class MavenLocalRepoResolveIntegrationTest extends AbstractDependencyResolutionT
         hasArtifact(moduleA)
     }
 
-    @Requires(TestPrecondition.JDK8_OR_EARLIER) //modifies environment variables
+    @Requires(TestPrecondition.SET_ENV_VARIABLE)
```

Comment:
Is this supposed to be part of this pull request? I'm guessing not.

Reply:
It is part of it, I'm adjusting both the tests and the docs to the expected behavior to be able to close #5187